pom.xml

```diff
@@ -31,23 +31,23 @@
         <dependency>
             <groupId>com.avispl.symphony.api</groupId>
             <artifactId>symphony-api-commons</artifactId>
-            <version>5.3.0</version>
```

Comment:
please update httpclient, junit versions, that provoke the dependabot warnings, also please align other dependency versions based on "3rd Party Software - Symphony" confluence page